tools/clang/test/CodeGenHLSL/container/SimpleBezier11DS.hlsl

```diff
@@ -0,0 +1,212 @@
+// RUN: %dxc -E main -T ds_6_0 %s | FileCheck %s
```

Comment:
Copied from MiniEngine because it's both run as batch test and used by container tests.